src/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './mysass.scss';
+import PersonalInfo from './PersonalInfo';
+
+class App extends React.Component {
+ 
+
+  render() {
+    return (
+      <div>
+        <div className="style">
+          <h1>Hello!</h1>
+          <p>This is my first Saas project!.</p>
+        </div>
+        {/* reuse component */}
+        <div className="validation">   
+          <h2 className="reuse">Component Reuse</h2>      
+          <PersonalInfo lable = "First" />
+          <PersonalInfo lable = "Second" />
+          <button>Submit</button>
+        </div>
+
+        <div>
+          <h1 className="nesting">Nesting Example</h1>
+          <nav>
```

Comment:
The danger of styling the html element directly is the moment this component is rendered in an html page all the html elements will be styled accordingly. This leads to lots of unintended styles.

So always avoid styling html elements directly

```suggestion
          <nav className="navigation">
```

```scss
.navigation {
  ul {
  }

  li {
  }
}
```
is fine